Corona-Warn-App/src/test/java/de/rki/coronawarnapp/datadonation/analytics/modules/keysubmission/AnalyticsKeySubmissionCollectorTest.kt

```diff
@@ -1,3 +1,210 @@
 package de.rki.coronawarnapp.datadonation.analytics.modules.keysubmission
 
-class AnalyticsKeySubmissionCollectorTest
+import de.rki.coronawarnapp.datadonation.analytics.storage.AnalyticsSettings
+import de.rki.coronawarnapp.risk.RiskLevelResult
+import de.rki.coronawarnapp.risk.RiskLevelSettings
+import de.rki.coronawarnapp.risk.RiskState
+import de.rki.coronawarnapp.risk.storage.RiskLevelStorage
+import de.rki.coronawarnapp.util.TimeStamper
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runBlockingTest
+import org.joda.time.Hours
+import org.joda.time.Instant
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+import testhelpers.preferences.mockFlowPreference
+
+class AnalyticsKeySubmissionCollectorTest : BaseTest() {
+
+    @MockK lateinit var timeStamper: TimeStamper
+    @MockK lateinit var analyticsSettings: AnalyticsSettings
+    @MockK lateinit var analyticsKeySubmissionStorage: AnalyticsKeySubmissionStorage
+    @MockK lateinit var riskLevelStorage: RiskLevelStorage
+    @MockK lateinit var riskLevelSettings: RiskLevelSettings
+    @MockK lateinit var riskLevelResult: RiskLevelResult
+
+    private val now = Instant.now()
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+        every { timeStamper.nowUTC } returns now
+    }
+
+    @Test
+    fun testReportTestRegistered() {
```

Comment:
very minor: (In this project) it's best practice to name test functions as follows w/o test prefix (given by annotation):

```
fun `ensures registered test is reported`() { }
```